app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_likes_counter
+    post.likes_counter += 1
+  end
+end
```

Comment:
- [ ] [OPTIONAL] As I have recommended above, I would like to again suggest that you can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```ruby
after_save :update_likes_counter

  private

  def increment_like_counter
   Code here ...
  end
```

Reply:
**Thank you Mr @Whoistolu for giving your time, I will add your suggestions.**